cluster/selfplay/cc_wrapper.sh

```diff
@@ -24,18 +24,19 @@ echo board_size: $BOARD_SIZE
 
 #gcloud auth activate-service-account --key-file=/etc/credentials/service-account.json
 echo Retrieiving Model
-MODEL_NAME=`gsutil ls "gs://$BUCKET_NAME/models/*.pb" | sort | tail -n 1`
+MODEL_FILE=`gsutil ls "gs://$BUCKET_NAME/models/*.pb" | sort | tail -n 1`
 echo Retrieiving games
-GAMES=`gsutil ls "gs://$BUCKET_NAME/data/selfplay/$MODEL_NAME/*.zz" | wc -l`
-
-gsutil cp $MODEL_NAME .
-NAME=`echo $MODEL_NAME  | rev | cut -d/ -f1 | rev`
+NAME=`echo $MODEL_FILE  | rev | cut -d/ -f1 | rev`
 BASENAME=`echo $NAME | cut -d. -f1`
+echo "gsutil ls 'gs://$BUCKET_NAME/data/selfplay/$BASENAME/*.zz'"
+GAMES=`gsutil ls "gs://$BUCKET_NAME/data/selfplay/$BASENAME/*.zz" | wc -l`
+
+gsutil cp $MODEL_FILE .
 
 mkdir -p data/selfplay
 mkdir -p sgf
 
-if [ $GAMES -lt 15000 ];
+if [ $GAMES -lt 25000 ];
```

Comment:
I'd suggest making this a constant at the top of the file and printing it on line 51